src/it/scala/resources/hive-site.xml

```diff
@@ -0,0 +1,17 @@
+<configuration>
```

Comment:
No sure we need this file? Where was this copied from?

Reply:
I just copied the resources from test, I wasn't sure if they were important I'll delete it